src/dcmri/ui_liver.py

```diff
@@ -21,6 +21,8 @@ class Liver(ui.Model):
           see :ref:`liver-tissues`. Defaults to 'UE'.
         sequence (str, optional): imaging sequence. Possible values are 'SS'
           and 'SR'. Defaults to 'SS'.
+        config (str, optional): configuration option for using pre-defined
+          variable values from use cases. Defaults to None.
```

Comment:
At this point can you please add the available options for the config parameter? Just 'TRISTAN-rat' for now